spec/acceptance/bundle_management_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper_acceptance'
+require 'fileutils'
+
+describe 'Managing Gemfile dependencies' do
+  before(:all) do
+    # Do everything in a clean tmpdir.
+    @old_pwd = Dir.pwd
+    @tmpdir = Dir.mktmpdir('pdk-acceptance')
+
+    Dir.chdir(@tmpdir)
+
+    # We need a module to test inside of.
+    shell_ex("#{path_to_pdk} new module test_module --skip-interview")
+  end
+
+  after(:all) do
+    Dir.chdir(@old_pwd)
+    FileUtils.remove_entry_secure(@tmpdir, true)
+  end
+
+  context 'when there is no Gemfile.lock' do
+    before(:all) do
+      shell_ex('rm -f test_module/Gemfile.lock')
+    end
+
+    after(:all) do
+      shell_ex('rm -f test_module/Gemfile.lock')
+    end
+
+    # TODO: come up with a way to invoke only the bundler stuff without trying to run unit tests
+    describe command("cd test_module && #{path_to_pdk} test unit") do
```

Comment:
This fails on powershell as && is not an operator there.
I think the only command that works on both Linux and PS is "cd test_module; #{path_to_pdk} test unit"

That comes with the weakness of always running command B even if command A fails.
There are [workarounds](https://stackoverflow.com/questions/11104197/chain-commands-in-powershell-command-line) to have Powershell fail and not run command B; but it would mean a command that doesn't work in Linux.

_Maybe_ we could have a helper to hide the platform specifics, like

    describe command("cd test_module").followed_by_command.(#{path_to_pdk} test unit") do

?